components/layouts/dashboard/dashboardDrawer/dashboardDrawerItem/DashboardDrawerItem.stories.tsx

```diff
@@ -0,0 +1,24 @@
+// import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { ComponentMeta } from '@storybook/react';
+// import DashboardDrawerItem, { IDashboardDrawerItem } from './DashboardDrawerItem';
+import DashboardDrawerItem from './DashboardDrawerItem';
+// import { defaultDashboardDrawerItemProps } from './DashboardDrawerItem.mocks';
+
+export default {
+    title: 'layouts/dashboard/dashboardDrawer/dashboardDrawerItem/DashboardDrawerItem',
+    component: DashboardDrawerItem,
+    // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+    argTypes: {},
+} as ComponentMeta<typeof DashboardDrawerItem>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+// const Template: ComponentStory<typeof DashboardDrawerItem> = (args) => (
+//     <DashboardDrawerItem {...args} />
+// );
+//
+// export const SDashboardDrawerItem = Template.bind({});
+// // More on args: https://storybook.js.org/docs/react/writing-stories/args
+//
+// SDashboardDrawerItem.args = {
+//     ...defaultDashboardDrawerItemProps.base,
```

Comment:
There are still args that will be useful to have in story books you might have to split the type in to two or define a new type in stories for the args that you can acctuly switch in storybooks. And then replace IDashboardDrawerItem